src/lib/KeyShares/KeySharesData/KeySharesPayload.ts

```diff
@@ -26,6 +26,8 @@ export class KeySharesPayload implements IKeySharesPayload {
         data.encryptedShares.map((share: EncryptShare) => share.publicKey),
         data.encryptedShares.map((share: EncryptShare) => share.privateKey)
       ),
+      encryptedKeys: data.encryptedShares.map((share: EncryptShare) => share.privateKey),
+      publicKeys: data.encryptedShares.map((share: EncryptShare) => share.publicKey),
```

Comment:
First of all this is duplicated, it already exists in nested data field. Secondly, to put it in a root level we should have product confirmation. And even after that, there should be additional changes to read it from a proper place when reading keyshares file

Better to remove redundant lines and keep as is